src/backend/libc/thread/syscalls.rs

```diff
@@ -418,12 +420,12 @@ pub(crate) fn setresgid_thread(
 // TODO: This could be de-multiplexed.
```

Comment:
This PR implements this TODO comment, so it can be removed.

Reply:
Done